### ISSUE TITLE
Error processing request when trying to 'Scan for Changes' via Infuse for iPad

### ISSUE THREAD

**Issue by tmechen** [jellyfin/jellyfin#4835]
**Describe the bug**
* "Scan for Changes" via Infuse 6.5.6 (most recent) on iPad fails with the below error in jellyfin logs
* playing (already indexed) files still work
* iPad is shown as active client in the jellyfin dashboard

**System (please complete the following information):**
 - OS: Linux 5.4.0-56-generic x86_64
 - Clients: iPad Infuse 6.5.6
 - Jellyfin Version: 10.6.4

**To Reproduce**

1. Connect Infuse 6 to Jellyfin Server
2. Click on "Scan for Changes"
3. See error

**Expected behavior**
* Scanning the library for new files

**Logs**
``` 
[2020-12-20 17:23:39.443 +00:00] [ERR] Error processing request. URL: "http://192.168.178.2:8096/InfuseSync/Checkpoint/31d9e44a4e8649f080451c38657fd1a3/UpdatedItems?IncludeItemTypes=Season&Fields=CommunityRating,DateCreated,Etag,Genres,MediaSources,OfficialRating,Overview,ParentId,Path,People,ProviderIds,PremiereDate,SortName&StartIndex=0&Limit=200"
System.NullReferenceException: Object reference not set to an instance of an object.
   at MediaBrowser.Controller.Entities.TV.Season.GetItemsInternal(InternalItemsQuery query)
   at MediaBrowser.Controller.Entities.Folder.FillUserDataDtoValues(UserItemDataDto dto, UserItemData userData, BaseItemDto itemDto, User user, DtoOptions fields)
   at Emby.Server.Implementations.Library.UserDataManager.GetUserDataDto(BaseItem item, BaseItemDto itemDto, User user, DtoOptions options)
   at Emby.Server.Implementations.Dto.DtoService.AttachUserSpecificInfo(BaseItemDto dto, BaseItem item, User user, DtoOptions options)
   at Emby.Server.Implementations.Dto.DtoService.GetBaseItemDtoInternal(BaseItem item, DtoOptions options, User user, BaseItem owner)
   at Emby.Server.Implementations.Dto.DtoService.GetBaseItemDtos(IReadOnlyList`1 items, DtoOptions options, User user, BaseItem owner)
   at InfuseSync.API.InfuseSyncItems.Get(GetUpdatedItemsQuery request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName)
   at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost httpHost, Object requestDto, IRequest req)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
[2020-12-20 17:23:39.444 +00:00] [WRN] HTTP Response 500 to "192.168.178.141". Time (slow): 0:00:01.0387957. "http://192.168.178.2:8096/InfuseSync/Checkpoint/31d9e44a4e8649f080451c38657fd1a3/UpdatedItems?IncludeItemTypes=Season&Fields=CommunityRating,DateCreated,Etag,Genres,MediaSources,OfficialRating,Overview,ParentId,Path,People,ProviderIds,PremiereDate,SortName&StartIndex=0&Limit=200"
```

**Additional context**
* found some discussion on reddit (https://www.reddit.com/r/jellyfin/comments/k7io6g/infuse_jellyfin_issues/) about problems with the http requests, but they are talking about current nightly build 10.7 and these look like are different problems
* didnt find anything else

**Comment by anthonylavado** [comment]
Have you installed the InfuseSync plugin on the server? It was thought to be optional, but it might even be required at this point.

**Comment by tmechen** [comment]
yupp, InfuseSync 1.2.4.0 is installed (might be default, i dont remember installing it myself)

i "fixed" it temporarly via re-adding the jellyfin server to infuse. 
also: while i could use it without any problems on my iphone, the same setup throw the errors for the ipad client
at the moment i cannot verify if the problem with the ipad persists, will update in some days

**Comment by anthonylavado** [comment]
This should be fixed from a little while ago now, as the 10.7.x has gone out, along with the updated plugin. Firecore has made some fixes too.

**Comment by dotslasher** [comment]
the same thing just happened to me using infuse Version 7.3.6 (7.3.4043) and jellyfin 10.7.7
